backend/ShootTheMoon/ShootTheMoon/Network/ShootServer.cs

```diff
@@ -102,6 +102,9 @@ public virtual async void OnNext(GameEvent info)
 
         private static async Task SendNotification(RpcClient client, Notification notification) {
             bool sent = false;
+            const int MAX_RETRIES = 10;
```

Comment:
Might want to move this to an env var at some point so we can set it when the container is spun up for testing

Reply:
Agreed, this would be good to pull from an environment variable with a default so that we can override if we want.